src/components/helpers/sidebar.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Side note: we need to either switch away from styled components or embrace it over css modules. That's a discussion we should probably have as a team, but I only now noticed that we are using both (see below)

Reply:
Ya we could talk about that soon. I am in favor of either one really.

Reply:
In the future, now that `App` is only the `Routes` component, it might be worth looking to get rid of it, but that can be a future story.

Reply:
ya i can add a story